lib/three_scale/metrics/yabeda.rb

```diff
@@ -32,10 +32,10 @@ def install!
               labels = {
                 controller: event.payload[:params]["controller"],
                 status: ThreeScale::Metrics::Yabeda.event_status_code_class(event),
-              }.merge!(event.payload.slice(*::Yabeda.default_tags.keys))
+              }
```

Comment:
Don't we need this for `rails_requests_total.increment(labels)`?

Reply:
We actually don't because this is only used when adding explicitly `default_tag :whatever`, and we don't do this, so `Yabeda.default_tags.keys` is always empty.